Dashboard/Dashboard/Presentation/PrimaryCourseDashboardView.swift

```diff
@@ -328,6 +331,34 @@ public struct PrimaryCourseDashboardView<ProgramView: View>: View {
                             .foregroundColor(Theme.Colors.textPrimary)
                             .accessibilityIdentifier("courses_header_text")
                         Spacer()
+                        Button(action: {
```

Comment:
@saeedbashir Maybe we should check `pushNotificationsEnabled` setting to show this button?